### ISSUE TITLE
Wordwide Targeting Error

### ISSUE THREAD

**Issue by pbarnhart** [Brainlabs-Digital/Google-Ads-Scripts#11]
Thanks for the script.  However, I was suprised when I found a number of accounts with targeting errors.  On checking the campaigns in question, I found that they did, in fact, have targeting.  But only radius targeting.

If a campaign has only radius targeting, your script misses it.  Not sure that this is the best way to solve this issue, but the following modification catches campaigns that have only radius targeting:

```
while (campaignIterator.hasNext()) {
var campaign = campaignIterator.next();
        var locationIterator = campaign
            .targeting()
            .targetedLocations()
            .get();
        if (!locationIterator.hasNext()) {
          // check if using only radius targeting
          var proximityIterator = campaign.targeting().targetedProximities().get();
          if (!proximityIterator.hasNext()) {
            var issue = {};
            issue[CAMPAIGN_NAME] = campaign.getName();
            worldwideTargetingIssues.push(issue);
          }
       }
    }
```

There may be a cleaner way to do this, but adding in the targetedProximities() in the loop corrected the problem for me.

**Comment by pbarnhart** [comment]
Actually spoke too soon.  Script still fails to detect radius targeting if the targeting locations exclusively uses "Google My Business locations."

**Comment by benm-brainlabs** [comment]
Hi Phil. Thanks for getting in touch! 

That looks like a great addition to the script. Would you like to make the Pull Request? Otherwise I'm happy to add it.

Have you got any thoughts on Google My Business locations?

**Comment by pbarnhart** [comment]
I will do the pull request today, thanks.  I have an idea re gmb but its
messy - if I get it to work I will do the pull request and add it.

I am bringing up the GMB issue with my staff today. Because of how GMB
locations extensions work, I want to see what happens when access
permission is removed on the GMB side to targeting.  I had set a policy to
always have the location zip code entered along with the gmb because of
this for my team, but mostly due to paranoia and no facts.

PS we run campaigns for about 100 accounts and after converting the script
to run at the MCC level caught several bid problems and two campaigns one
of my team set up last week that were set to worldwide!

Want the MCC level script too?

**Comment by benm-brainlabs** [comment]
It's great that we were able to help!

Can you make the Pull Request just the addition of the code above? (I've already taken a look and I think we can merge it in pretty much straight away.)

**Comment by pbarnhart** [comment]
Sure thing.

**Comment by benm-brainlabs** [comment]
Merged in with #12.
